__tests__/app.test.js

```diff
@@ -233,4 +233,93 @@ describe('GET /api/reviews/:review_id', () =>
             });
         });
     });
+});
+
+describe('GET /api/reviews/:review_id/comments', () =>
+{
+    test('server responds with 200 status code', () =>
+    {
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200);
+    });
+
+    test('server responds with an array of comment objects for given review_id', () =>
+    {
+        return request(app)
+        .get('/api/reviews/3/comments')
+        .then((response) =>
+        {
+            const { comments } = response.body;
+
+            expect(comments).toHaveLength(3);
+
+            for(let comment of comments)
+            {
+                expect(comment.review_id).toBe(3);
+
+                expect(comment).toHaveProperty('comment_id');
```

Comment:
We can check the values for the keys too to ensure theyre the correct type, look at jest "expect.any()"
and we can test the id for the exact value, as we have the value in the URL

Reply:
sure, added expect.any() to all the property values.